Yearn-agents/PPSValues/src/mock.ts

```diff
@@ -0,0 +1,34 @@
+import { createAddress } from "forta-agent-tools";
+
+const mockPrice = jest.fn();
+mockPrice
+  .mockReturnValueOnce(1.1) // First call. Tracker = 101
+  .mockReturnValueOnce(1) // Second test. Tracker = 100  (<101)
+  .mockReturnValueOnce(100) // Third test. Tracker = 200. ( > 100 )
+  .mockReturnValueOnce(10); // Fourth test.
```

Comment:
is it possible to call this mock return value in the test case itself so it is clear what the price changed to in the test case?

Reply:
the jest.fn() is defined inside the class. Best i can do is export this and rewrite it in the tests.